draft-ietf-quic-tls.md

```diff
@@ -656,13 +655,14 @@ all the acknowledgements necessary to reach the same state.
 
 After all CRYPTO frames for a given encryption level have been sent and all
 expected CRYPTO frames received, and all the corresponding acknowledgments have
-been received or sent, an endpoint starts a timer.  To limit the effect of
-packet loss around a change in keys, endpoints MUST retain packet protection
-keys for that encryption level for at least three times the current
-Retransmission Timeout (RTO) interval as defined in {{QUIC-RECOVERY}}.
-Retaining keys for this interval allows packets containing CRYPTO or ACK frames
-at that encryption level to be sent if packets are determined to be lost or new
-packets require acknowledgment.
+been received or sent, an endpoint starts a timer.  For 0-RTT keys, which do not
+carry CRYPTO frames, this timer starts when the first packets protected with
```

Comment:
The 0-RTT timer can be much more aggressive than 3 * RTO, since nothing now ever needs to be retransmitted at 0-RTT, it can always be sent with 1-RTT keys instead.

Reply:
I don't think that we need to special case that - as we agreed, there are optimizations that are possible, but we don't need to describe them.  The trick here is that the sender/client might stop using 0-RTT, but the receiver/server can't as easily.